### PR TITLE
v63.0.0

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
Ready for review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,178 @@
...
+
+* [**Breaking**:  Add a card shape themer (#5031)](https://github.com/material-components/material-components-ios/commit/c036e22a7dde0c2c970e380123334f870cdabaae) (Yarden Eitan)
+
+This is a breaking change due to the addition of the `shapeScheme` property to the MDCCardScheming
+protocol. If you have created a type that conforms to MDCCardScheming you will need to implement the
+`shapeScheme` property now as well.
+
+## New features
+
+ActionSheet backgroundColor can now be customized.
```
Comment (romoore) [inline CHANGELOG.md]: Write more clearer: Please consider making these an unordered list.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,178 @@
...
+
+BottomAppBar has a new surface variant color themer. Example:
```
Comment (romoore) [inline CHANGELOG.md]: Maybe leave the example out of the CHANGELOG and link to docs instead?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,178 @@
...
+
+### BottomNavigation
+
+#### MDCBottomNavigationBar
+
+*new* method: `-viewForItem:` in `MDCBottomNavigationBar`
+
+### BottomSheet+ShapeThemer
+
+**New component.**
```
Comment (romoore) [inline CHANGELOG.md]: Nit: This isn't a component. s/component/themer/

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,178 @@
...
+
+*new* property: `shapeScheme` in `MDCCardScheme`
+
+#### MDCCardScheming
+
+*new* property: `shapeScheme` in `MDCCardScheming`
+
+### Cards+ShapeThemer
+
+**New component.**
```
Comment (romoore) [inline CHANGELOG.md]: Nit: This isn't a component. s/component/themer/

**Review by romoore — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (jverkoey) [reply CHANGELOG.md]: Filed https://github.com/material-components/material-components-ios/issues/5105

File: CHANGELOG.md (hunk 4 above)
Comment (jverkoey) [reply CHANGELOG.md]: Filed https://github.com/material-components/material-components-ios/issues/5105

Comment (romoore) [reply CHANGELOG.md]: Would you please fix it manually for this release? No sense being inaccurate just because of a bug in our tooling.

Comment (romoore) [reply CHANGELOG.md]: Whoops, you did. I didn't realize this was a resolved conversation. Sorry!

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,169 @@
...
+
+* [**Breaking**:  Add a card shape themer (#5031)](https://github.com/material-components/material-components-ios/commit/c036e22a7dde0c2c970e380123334f870cdabaae) (Yarden Eitan)
+
+This is a breaking change due to the addition of the `shapeScheme` property to the MDCCardScheming
+protocol. If you have created a type that conforms to MDCCardScheming you will need to implement the
+`shapeScheme` property now as well.
+
+## New features
+
+1. ActionSheet's `backgroundColor` can now be customized.
```
Comment (romoore) [inline CHANGELOG.md:19]: Nit: Please use an unordered list here. These items don't have a natural ordering so I think using an unordered list is more appropriate.

**Pull request merged**